tests/garage/torch/modules/test_mlp_module.py

```diff
@@ -0,0 +1,48 @@
+import pickle
+import torch
+import unittest
+
+from nose2.tools.params import params
+import numpy as np
+import torch.nn as nn
+
+from garage.torch.modules import MLPModule
+
+
+class TestMLPModel(unittest.TestCase):
+    @params((5, 1, (1, )), (5, 1, (2, )), (5, 2, (3, )), 
+            (5, 2, (1, 1)), (5, 3, (2, 2)))
+    def test_output_values(self, input_dim, output_dim, hidden_sizes):
+        input_val = torch.ones([1, 5], dtype=torch.float32)
+        module = MLPModule(
+            input_dim=input_dim,
+            output_dim=output_dim,
+            hidden_sizes=hidden_sizes,
+            hidden_w_init=nn.init.ones_,
+            output_w_init=nn.init.ones_)
+        output = module.forward(input_val)
+
+        expected_output = torch.full([1, output_dim], 
+                                     fill_value=5 * np.prod(hidden_sizes),
+                                     dtype=torch.float32)
+        
+        self.assertEqual(torch.all(torch.eq(output, expected_output)), True)
+
+    @params((5, 1, (1, )), (5, 1, (2, )), (5, 2, (3, )), 
+            (5, 2, (1, 1)), (5, 3, (2, 2)))
+    def test_is_pickleable(self, input_dim, output_dim, hidden_sizes):
+        input_val = torch.ones([1, 5], dtype=torch.float32)
+        module = MLPModule(
+            input_dim=input_dim,
+            output_dim=output_dim,
+            hidden_sizes=hidden_sizes,
+            hidden_w_init=nn.init.ones_,
+            output_w_init=nn.init.ones_)
+        output1 = module.forward(input_val)
+
+        h = pickle.dumps(module)
+        model_pickled = pickle.loads(h)
+        output2 = model_pickled.forward(input_val)
+
+        assert np.array_equal(torch.all(torch.eq(output1, output2)), True)
+        
```

Comment:
note the missing newline. the CI will ding you on this.

you can configure your editor to add these (also to strip trailing whitespace, use spaces instead of tabs, etc.). see CONTRIBUTING.md for details. there is a .editorconfig if you'd like to use that plug-in.

Reply:
Oh right I totally forgot about that. Just installed the hooks